src/components/ListedSpecies/SexSection.jsx

```diff
@@ -0,0 +1,58 @@
+import {
+  FormControl,
+  FormLabel,
+  Grid,
+  GridItem,
+  Input,
+  NumberDecrementStepper,
+  NumberIncrementStepper,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+} from '@chakra-ui/react';
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import CollapsibleSection from '../CollapsibleSection/CollapsibleSection';
+
+const inputs = [
+  '# of Male Adults',
+  '# of Male Fledges',
+  '# of Male Chicks',
+  '# of Female Adults',
+  '# of Female Fledges',
+  '# of Female Chicks',
+];
+
+const SexSection = () => {
+  const { register, setValue, getValues } = useFormContext();
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'register' is assigned a value but never used.